install.sh

```diff
@@ -419,7 +430,7 @@ install_npm() {
   echoh
   echoerr "Please install npm or yarn to install code-server!"
   echoerr "You will need at least node v12 and a few C dependencies."
-  echoerr "See the docs https://github.com/cdr/code-server/blob/v3.10.2/docs/install.md#yarn-npm"
+  echoerr "See the docs https://coder.com/docs/code-server/v3.10.2/install#yarn-npm"
```

Comment:
Are you planning to bump the version number each release, or should we link to `/latest/` here?

Reply:
> If so, I think we should replace latest with the version number in this case actually so that old docs will link to the correct branch.

I think @bpmct and @BrunoQuaresma thought we should use the version ([source](https://github.com/cdr/code-server/pull/3684#discussion_r659870342)).

We do have a script called `release-prep.sh` which handles updating these links for us. See [here](https://github.com/cdr/code-server/blob/main/ci/build/release-prep.sh#L81-L84)

Reply:
Yes, we want to keep the version so the script `release-prep` handle the update for us.